pilot/cmd/pilot-agent/options/options.go

```diff
@@ -106,6 +106,12 @@ var (
 		"Envoy health status port value").Get()
 	envoyPrometheusPortEnv = env.RegisterIntVar("ENVOY_PROMETHEUS_PORT", 15090,
 		"Envoy prometheus redirection port value").Get()
+	tdInterceptionPortEnv = env.RegisterStringVar("TRAFFICDIRECTOR_INTERCEPTION_PORT","15001",
+		"TrafficDirector Interception port value")
+	tdInboundInterceptionPortEnv = env.RegisterStringVar("TRAFFICDIRECTOR_INBOUND_INTERCEPTION_PORT", "15006",
+		"TrafficDirector Inbound Interception port value")
+	tdInboundBackendPortsEnv = env.RegisterStringVar("TRAFFICDIRECTOR_INBOUND_BACKEND_PORTS", "8000",
+		"TrafficDirector Inbound Backend ports value")
```

Comment:
I think it is better not to expose these special options to normal users

Reply:
Ack, removed this.